_src/scroll.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const element = document.querySelector('navigation .active');
```

Comment:
Instead of auto-scrolling the active menu item to the top of the window, could we scroll it to the centre?

Reply:
This is what should happen on most browsers: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoViewIfNeeded#browser_compatibility

What browser are you seeing this on?

Reply:
I'm using Chrome, and the active menu item is being scrolled to the top of the window.